sdk/containerregistry/container-registry/review/container-registry.api.md

```diff
@@ -67,7 +67,7 @@ export class ContainerRegistryBlobClient {
     readonly endpoint: string;
     readonly repositoryName: string;
     uploadBlob(blobStream: NodeJS.ReadableStream, options?: UploadBlobOptions): Promise<UploadBlobResult>;
-    uploadManifest(manifest: (() => NodeJS.ReadableStream) | NodeJS.ReadableStream | OciManifest, options?: UploadManifestOptions): Promise<UploadManifestResult>;
+    uploadManifest(manifest: NodeJS.ReadableStream | OciManifest, options?: UploadManifestOptions): Promise<UploadManifestResult>;
```

Comment:
Do you have the equivalent of BinaryData that .NET and JS have?  We have been adding overloads to take these (but if you don't have it in JS, it wouldn't be relevant to you).

Reply:
We have `Buffer` which is a Node builtin type (not Azure SDK specific like BinaryData). It is very easy to convert `Buffer` to a ReadableStream. I will add the overload.